Sources/SwiftGenKit/Parsers/Fonts/CTFont.swift

```diff
@@ -5,27 +5,87 @@
 //
 
 import AppKit.NSFont
+import CoreText
 import Foundation
 import PathKit
 
 extension CTFont {
-  static func parse(file: Path, relativeTo parent: Path? = nil) -> [Fonts.Font] {
+  static func parse(file: Path, relativeTo parent: Path? = nil, extractCodePoints: Bool = false) -> [Fonts.Font] {
     let descs = CTFontManagerCreateFontDescriptorsFromURL(file.url as CFURL) as NSArray?
     guard let descRefs = (descs as? [CTFontDescriptor]) else { return [] }
 
+    // swiftlint:disable:next closure_body_length
     return descRefs.compactMap { desc -> Fonts.Font? in
       let font = CTFontCreateWithFontDescriptorAndOptions(desc, 0.0, nil, [.preventAutoActivation])
       let postScriptName = CTFontCopyPostScriptName(font) as String
       guard let familyName = CTFontCopyAttribute(font, kCTFontFamilyNameAttribute) as? String,
         let style = CTFontCopyAttribute(font, kCTFontStyleNameAttribute) as? String else { return nil }
 
       let relPath = parent.flatMap { file.relative(to: $0) } ?? file
+
+      let characterSet = CTFontCopyCharacterSet(font) as CharacterSet
+      let cgFont = CTFontCopyGraphicsFont(font, nil)
```

Comment:
Why not put these 2 inside the `if extractCodePoints`?
That might not take _that_ long to run (though maybe that does depending on the font file size?) but better avoid needless calls when the flag is off 🙃

Reply:
Also, not completely sure because maybe the bridging of that API to Swift takes care of that, but worth double-checking: since those two calls are `…Copy…` don't they need a balancing `CTRelease` (or `CFRelease`?) call or similar to avoid leaks?